chain/types/ethtypes/eth_legacy_homestead_transactions_test.go

```diff
@@ -126,15 +127,6 @@ func TestLegacyHomesteadSignatures(t *testing.T) {
 			"",
 			true,
 		},
-		{
-			"0xf86f830131cf8504a817c800825208942cf1e5a8250ded8835694ebeb90cfa0237fcb9b1882ec4a5251d1100008026a0f5f8d2244d619e211eeb634acd1bea0762b7b4c97bba9f01287c82bfab73f911a015be7982898aa7cc6c6f27ff33e999e4119d6cd51330353474b98067ff56d930",
```

Comment:
But this particular serialised tx has a `ChainID` of 1 in it's `V` value so it will be parsed as a EIP-155 tx.

The way we differentiate b/w EIP-155 & Homestead transactions is simply by looking at the `V` value and checking if it is 27 or 28.

Reply:
https://rawtxdecode.in/